src/workloads/query/SlidingWindows.yml

```diff
@@ -127,6 +129,19 @@ Actors:
               $derivative: {input: "$temp", unit: "second"},
               window: {documents: [-500, 0]}}}}}]
         cursor: {batchSize: *batchSize}
+  - *Nop
+
+
+- Name: SlidingWindows
```

Comment:
Why does separating to a new phase work?

Reply:
I did it to not contaminate the operation throughput of earlier existing tests. Apparently the operation throughput is the number of operations per unit of phase time and not per unit of that operation time.